experiments/long_runs/leaderboard/data_sources.jl

```diff
@@ -0,0 +1,151 @@
+import ClimaAnalysis
+
+# To add a variable for the leaderboard, add a key-value pair to the dictionary
+# `sim_var_dict` whose key is the short name of the variable and the value is a function
+# that returns a `OutputVar` Any preprocessing is done in the function which includes unit
+# conversion and shifting the dates.
+#
+# Then, add a key-value pair to the dictionary `obs_var_dict` whose key is the same short
+# name as before and the value is a function that takes in a start date and returns a
+# `OutputVar`. Any preprocessing is done in the function.
+#
+# Next, add a key-value pair to the dictionary `mask_dict` whose key is the same short name
+# as before and the value is a function that takes in a OutputVar representing simulation
+# data and a OutputVar representing observational data and returns a masking function or
+# `nothing` if no masking function is needed. The masking function is used to correctly
+# normalize the global bias and global RMSE.
+#
+# Finally, add a key-value pair to the dictionary `compare_vars_biases_plot_extrema` whose
+# key is the same short name as before and the value is a tuple of floats which determine
+# the range of the bias plots.
+
+# For plotting bias plots
+compare_vars_biases_plot_extrema = Dict(
+    "et" => (-0.00001, 0.00001),
+    "gpp" => (-8.0, 8.0),
+    "lwu" => (-40.0, 40.0),
+)
+
+# Dict for loading in simulation data
+sim_var_dict = Dict{String, Any}()
+
+for short_name in ["et", "lwu"]
+    sim_var_dict[short_name] =
+        () -> begin
+            sim_var = get(
+                ClimaAnalysis.SimDir(diagnostics_folder_path),
+                short_name = short_name,
+            )
+            # Remove the line below later when start_date is added to the diagnostics (see
+            # issue #879 in ClimaLand.jl)
+            haskey(sim_var.attributes, "start_date") ||
+                (sim_var.attributes["start_date"] = "2012-01-01T00:00:00")
+            sim_var =
+                ClimaAnalysis.shift_to_start_of_previous_month(sim_var)
+            return sim_var
+        end
+end
+
+for short_name in ["gpp"]
+    sim_var_dict[short_name] =
+        () -> begin
+            sim_var = get(
+                ClimaAnalysis.SimDir(diagnostics_folder_path),
+                short_name = short_name,
+            )
+            # Remove the line below later when start_date is added to the diagnostics (see
+            # issue #879 in ClimaLand.jl)
+            haskey(sim_var.attributes, "start_date") ||
+                (sim_var.attributes["start_date"] = "2012-01-01T00:00:00")
+            sim_var =
+                ClimaAnalysis.shift_to_start_of_previous_month(sim_var)
+            sim_var = ClimaAnalysis.convert_units(
+                sim_var,
+                "g m-2 day-1",
+                conversion_function = units -> units * 86400.0 * 12.011,
```

Comment:
What is 12.011?

Reply:
The units in the simulation is `mol CO2 m^-2 s^-1` which is converted to match the units in the observational data which is `g C m^-2 day^-1`. I multiplied by `(86400 s / 1 day)`, `44.009 g CO2 / 1 mol CO2`, and `12.011 g C / 44.009 g CO2` to convert the units. I am not sure if the unit conversion is right.

Reply:
This is correct